binding.cc

```diff
@@ -1015,6 +1025,86 @@ namespace zmq {
 
 #endif
 
+  NAN_METHOD(Socket::Readv) {
+    Socket* socket = GetSocket(info);
+    if (socket->state_ != STATE_READY)
+      return info.GetReturnValue().Set(Nan::Undefined());
+
+    int events;
+    size_t events_size = sizeof(events);
+
+    int rc = 0;
+    int flags = 0;
+    int64_t more = 1;
+    size_t more_size = sizeof(more);
+    size_t index = 0;
+
+    while (true) {
```

Comment:
to @kkoopa's point i prefer to use an `int` set to 1 for the flag. set it to zero to beak out